mirage/serializers/user.ts

```diff
@@ -5,13 +5,13 @@ import User from 'ember-osf-web/models/user';
 
 import { randomGravatar } from '../utils';
 
-import ApplicationSerializer from './application';
+import ApplicationSerializer, { SerializedRelationships } from './application';
 
 const { OSF: { apiUrl } } = config;
 
 export default class UserSerializer extends ApplicationSerializer<User> {
     buildRelationships(model: ModelInstance<User>) {
-        return {
+        const retVal: SerializedRelationships<User> = {
```

Comment:
minor: `retVal` is not a good variable name
```suggestion
        const serializedRelationships: SerializedRelationships<User> = {
```